src/components/dashboard/SendTokens/Bitcoin/Form.jsx

```diff
@@ -4,13 +4,18 @@
  */
 
 import { connect } from 'react-redux'
+import { reduxForm } from 'redux-form/immutable'
 import { estimateBtcFee } from '@chronobank/core/redux/bitcoin/thunks'
+
+import { FORM_SEND_TOKENS } from 'components/constants'
 import BitcoinLikeBlockchainForm, { mapStateToProps } from '../BitcoinLikeBockchain/Form'
+import validate from '../validate'
 
 function mapDispatchToProps (dispatch) {
   return {
     estimateFee: (params) => dispatch(estimateBtcFee(params)),
   }
 }
 
-export default connect(mapStateToProps, mapDispatchToProps)(BitcoinLikeBlockchainForm)
+const BitcoinReduxForm = reduxForm({ form: FORM_SEND_TOKENS, validate })(BitcoinLikeBlockchainForm)
```

Comment:
Please, use decorators

Reply:
Why? What are the benefits?

Reply:
If I try to use the decorator I get the following error

```
Syntax error: Leading decorators must be attached to a class declaration (21:0)

  19 | 
  20 | @reduxForm({ form: FORM_SEND_TOKENS, validate })
> 21 | const BitcoinReduxForm = AbstractBitcoinForm
     | ^
  22 | export default connect(mapStateToProps, mapDispatchToProps)(BitcoinReduxForm)
```

Reply:
try like this:
```
@connect(mapStateToProps, mapDispatchToProps)
@reduxForm({ form: FORM_2FA_WALLET })
export default class TwoFaWalletForm extends PureComponent {
 ```